SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.UWP/Crashes.cs

```diff
@@ -35,6 +35,23 @@ public void OnChannelGroupReady(IChannelGroup channelGroup, string appSecret)
 #if REFERENCE
 #else
                 WatsonRegistrationManager.Start(appSecret);
+                MobileCenter.CorrelationIdChanged += (s, id) =>
+                {
+                    // Checking for null and setting id needs to be atomic to avoid
+                    // overwriting. But only do that if the id argument is null to avoid
+                    // needlessly waiting on the lock.
+                    if (id == null &&
+                        MobileCenter.TestAndSetCorrelationId(null, Guid.NewGuid().ToString()))
```

Comment:
Ok I didn't get it. Why do we try to generate a new guid here?

Reply:
I can probably remove this.

Reply:
I can probably remove that. I had an explanation, but after thinking about it more, I think it's unnecessary.

Reply:
Let's simplify then.